config/cable.yml

```diff
@@ -6,5 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
-  channel_prefix: bostonrb-website_production
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
Cool - I've never passed a block to `ENV.fetch`. It doesn't look like we're using the return value in the block. What do you think of setting this as a default value instead? (ie: `ENV.fetch("REDIS_URL", "redis://localhost:6379/1"`)

Reply:
I just went with the rails-generated file here ... I'd be fine with either style. At a quick glance, it looks like the rails update generator is just always going with the block style even when there's just a static value to return (puma config has similar stuff).

Reply:
I just noticed that. Keeping it consistent works for me 👍